src/MerQure.RbMQ/SharedConnection.cs

```diff
@@ -8,7 +8,7 @@ namespace MerQure.RbMQ
     /// <summary>
     /// Must be registered as a singleton
     /// </summary>
-    public class SharedConnection
+    public class SharedConnection : IDisposable
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix this implementation of 'IDisposable' to conform to the dispose pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3881)